examples/training/semantic_segmentation/pascal_voc/fcn.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
```

Comment:
Can you update in the PR description the trained model quality? (meanIOU)

Reply:
Right. I haven't run the training script yet due to a lack of resources, but I will try to do so on Colaboratory and see what comes out of it

Reply:
Or are you looking for the reported results from the paper?

Reply:
Have updated the reported results in the issue description as well as the PR description

Reply:
He meant the performance of this PR's model - if you had the chance to run it on the training script which is included here.

Even though we're moving towards generalizing the training script, you can run it locally (or in an online environment) to validate the model's performance ahead of the review/merge

Reply:
My two cents: you can run it on Kaggle or Colab for ~10-15 epochs. Many semantic segmentation archs learn the majority of what they'll learn in the first epochs, with diminishing returns down the line. If it looks like it's showing promise there, it'll likely a correct implementation

Reply:
I suppose 10 epochs are sufficient.